unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/Settings/Settings.cs

```diff
@@ -110,6 +110,7 @@ private SettingsData.GeneralSettings GetDefaultGeneralSettings()
             {
                 sfxVolume = 1,
                 mouseSensitivity = 0.6f,
+                scenesLoadRadius = 10,
```

Comment:
Default value should be 4, not 10.

Reply:
in kernel master the LOS default is 10. 4 is the "secure radius":
https://github.com/decentraland/explorer/blob/444a95484ed484c5eaba47984aecbb19695d91bc/kernel/packages/decentraland-loader/lifecycle/manager.ts#L122

Reply:
Corrected.